src/auto/app_info.rs

```diff
@@ -1,9 +1,11 @@
-// This file was generated by gir (13f739b) from gir-files (469db10)
+// This file was generated by gir (5a68ad0) from gir-files (469db10)
 // DO NOT EDIT
 
 use AppInfoCreateFlags;
 use AppLaunchContext;
 #[cfg(any(feature = "v2_50", feature = "dox"))]
+use AsyncResult;
```

Comment:
To many of these added and unused :(

Reply:
That's because of a last check which removes some functions (the async ones). Therefore, the associated imports are still there.